### PR TITLE
Add midsommar story piece and adjust somna-om

### PR DESCRIPTION
**Pull request by UtvecklareAmber**
to include this option

### CODE REVIEW

**Comment by UtvecklareAmber** [comment]
Please review this, @casey-collab

File: swedish/somna-om/somna-om.md [hunk 1]
```diff
@@ -1,4 +1,5 @@
 Du väljer att dra täcket över huvudet och somnar om,
 men mardrömmen fortsätter och du drömmer nu om att...
 
-[du äter en stor marshmallow...](../marshmallowen/marshmallowen.md)
\ No newline at end of file
+[du äter en stor marshmallow...](../marshmallowen/marshmallowen.md)
+[det är midsommar](../midsommar/midsommar.md)
```
Comment (casey-collab) [inline swedish/somna-om/somna-om.md]: There is a link on this line and also on the previous line.  Please make sure to insert a blank line between links.  That will help keep options visually separated.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by UtvecklareAmber** [comment]
Maybe I need to ask @casey-collab to check again?

**Comment by casey-collab** [comment]
Looks great! :rabbit:

**Pull request merged**